tests/layer_tests/tensorflow_tests/test_tf_AddN.py

```diff
@@ -66,3 +64,49 @@ def test_addn_placeholder_const(self, params, ie_device, precision, ir_version,
                                                           use_legacy_frontend=use_legacy_frontend),
                    ie_device, precision, ir_version, temp_dir=temp_dir,
                    use_legacy_frontend=use_legacy_frontend)
+
+class TestComplexAddN(CommonTFLayerTest):
+    def _prepare_input(self, inputs_info):
+        rng = np.random.default_rng()
+        assert 'param_imag_1:0' in inputs_info
+        assert 'param_real_1:0' in inputs_info
+        assert 'param_real_2:0' in inputs_info
+        assert 'param_imag_2:0' in inputs_info
+        param_real_shape_1 = inputs_info['param_real_1:0']
+        param_imag_shape_1 = inputs_info['param_imag_1:0']
+        param_real_shape_2 = inputs_info['param_real_2:0']
+        param_imag_shape_2 = inputs_info['param_imag_2:0']
+        inputs_data = {}
+        inputs_data['param_real_1:0'] = 4 * rng.random(param_real_shape_1).astype(np.float32) - 2
+        inputs_data['param_imag_1:0'] = 4 * rng.random(param_imag_shape_1).astype(np.float32) - 2
+        inputs_data['param_real_2:0'] = 4 * rng.random(param_real_shape_2).astype(np.float32) - 2
+        inputs_data['param_imag_2:0'] = 4 * rng.random(param_imag_shape_2).astype(np.float32) - 2
+        return inputs_data
+
+    def create_complex_addn_net(self, input_shapes):
+        tf.compat.v1.reset_default_graph()
+        with tf.compat.v1.Session() as sess:
+            param_real_1 = tf.compat.v1.placeholder(np.float32, input_shapes[0], 'param_real_1')
+            param_imag_1 = tf.compat.v1.placeholder(np.float32, input_shapes[1], 'param_imag_1')
+            param_real_2 = tf.compat.v1.placeholder(np.float32, input_shapes[2], 'param_real_2')
+            param_imag_2 = tf.compat.v1.placeholder(np.float32, input_shapes[3], 'param_imag_2')
+            complex_1 = tf.raw_ops.Complex(real=param_real_1, imag=param_imag_1)
+            complex_2 = tf.raw_ops.Complex(real=param_real_2, imag=param_imag_2)
+            tf.raw_ops.AddN(inputs=[complex_1, complex_2], name='complex_AddN')
```

Comment:
please add `Real` and `Imag` operations after `AddN`. OV does not support complex tensors on output of a model. We support only for intermediate tensors.

Reply:
Thanks for the suggestion! I've implemented it. Know all tests passes with `FP 16`